src/img/banner.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ***** BEGIN LICENSE BLOCK *****
+  - Part of the Jellyfin project (https://jellyfin.media)
+  - 
+  - All copyright belongs to the Jellyfin contributors; a full list can
+  - be found in the file CONTRIBUTORS.md
+  - 
+  - This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License.
+  - To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/4.0/.
+- ***** END LICENSE BLOCK ***** -->
```

Comment:
Shall we remove this block?

We already have it in the original **jellyfin-ux** repo and we might save a few kilobytes.

Reply:
Sounds fine by me

Reply:
Licensing stuff is not my strong side though

Reply:
I'm not sure, I think someone on the general matrix chat mentioned that we should have licensing at the start of all the files. Hopefully someone with more knowledge of licensing can help.

Reply:
Ideally, the SVG files should be optimized during build, which would remove the licensing block anyway.

I don't know the build process for this repo, but I'm guessing it's packing everything with Webpack, so using something like [image-webpack-loader](https://github.com/tcoopman/image-webpack-loader), which uses `imagemin` (the same tool we use to optimize images on `jellyfin-web`) should do the trick.

Reply:
@MrTimscampi This sounds perfect! I'll investigate this.

Reply:
We should leave it, its an SVG which is a source file.

Reply:
@joshuaboniface Leaving and removing the blocks in production in 24ede02

Reply:
ShareAlike also mandates that any derivative is also properly licensed under the same license so it should be noted on the license/about page. Cause even the minified SVG is STILL CC-BY-SA.

Reply:
@EraYaN Adding a header at the bottom  of the LICENSE.md file with a copy-paste of the license included in the SVG is enough?

Sorry, I'm not a license expert

Reply:
You can pass options to the SVGO plugin of Imagemin to keep comments. The option in question is `removeComments` (`true` by default)

Reply:
Went with removeComments=false